datasets/loader_regression.py

```diff
@@ -196,9 +197,10 @@ def year_prediction_msd(dataset_dir: Path) -> bool:
     YearPredictionMSD dataset from UCI repository
     https://archive.ics.uci.edu/ml/datasets/yearpredictionmsd
 
-    TaskType:regression
-    NumberOfFeatures:90
-    NumberOfInstances:515345
+    year_prediction_msd x train dataset (463715, 11)
```

Comment:
Are you sure about 11 features? Do we have an error in this number now?
Can you please mark these datasets as `regression task` as before (line 71)

Reply:
Mistake fixed